draft-scenarios/clash/king_of_the_hill_v2.tex

```diff
@@ -0,0 +1,130 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{King of the Hill v2}{\images/kingofthehill.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} KABLAMIO \& Cantila
+
+\textbf{Source:} \href{https://discord.com/channels/1275490124301467799/1284428969369665631}{KABLAMIO's Discord}
+
+\textit{All eyes turn to the Hill — where legends rise and empires fall. Fight, claim it, and prove you’re worthy to rule.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This Scenario plays out over 6 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 -- 3
+
+\textbf{Starting Resources:} 22 \svg{gold}, 5 \svg{building_materials}
+
+\textbf{Starting Income:} 15 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few of cheapest \silver\ Units
+\end{itemize}
+
+\textbf{Town Buildings:}
+\begin{itemize}
+  \item \bronze\ Dwelling
+\end{itemize}
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item P × Far (II-III) Map Tiles
+  \item P × Near (IV-V) Map Tiles
+  \item Center Map Tile (C1 or C3)
+  \item Custom Center Map Tile
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Points}}
+After Round 6, the Player with the most Victory Points (VP) wins. Players get
+VPs for:
+\begin{itemize}
+  \item Round number - 1 VP for starting your turn on ``The Hill''
+  \item 2 VP for winning a Combat against another Player
+  \item 1 VP for losing a Combat against another Player
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\begin{itemize}
+  \item \textbf{\nth{2} Round:} Gain secondary Hero for free.
+  \item \textbf{\nth{3} Round:} Gain Citadel for free. If you had it, gain 1 VP.
+  \item \textbf{\nth{4} Round:} Gain \silver\ dwelling for free. If you had it: gain 2 VP.
+  \item \textbf{\nth{5} Round:} Gain \golden\ dwelling for free. If you had it: gain 3 VP.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+  \item Every Round after Round 1 is a Resource Round.
+  \item Players can't attack their opponent's starting Town.
+  \item Players can only attack another player once each turn.
+  \item Players do not lose gold for losing or retreating from any Combat.
+  \item Secondary Heroes act as Main Hero.
+  \item After losing a Combat, the defeated Hero is moved to it's starting Town.
+  \item After PvP Combat, both Players gain their lost \bronze\ and \silver\ Units back.
+  \item Players may \svg{pay_v2} 6 \svg{gold} and 6 \svg{building_materials} to gain Walls and Arrow Tower until their next turn.
+  \item Level VI and VII Combats cannot be skipped.
+  \item Level VII Neutral Combat reward: Gain 20 \svg{gold}.
```

Comment:
```suggestion
  \item Level VI and VII Combats cannot be skipped.
  \item ``The Hill'' Combat reward: 20 \svg{gold}.
```
The Hill is the only VII field, right? In that case, it could be explicit.